src/scripts/main.js

```diff
@@ -1,3 +1,101 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('#growth-table');
+  const appendRowButton = document.querySelector('.append-row');
+  const removeRowButton = document.querySelector('.remove-row');
+  const appendColumnButton = document.querySelector('.append-column');
+  const removeColumnButton = document.querySelector('.remove-column');
```

Comment:
Ensure that the elements with the IDs and classes used here (`#growth-table`, `.append-row`, `.remove-row`, `.append-column`, `.remove-column`) exist in your HTML. If any of these elements are missing, the script will not function as expected.